ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent.json

```diff
@@ -482,13 +482,6 @@
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "CRU"
   },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseEventID": "internal-changeState-Gatekeeping->PREPARE_FOR_HEARING",
```

Comment:
Is this ID safe to delete? I guess it will remove it from history tab for systemupdate user but thats probably fine? @akomisarek

Reply:
Safe to delete, we never care about history for the systemuser and if we don't need to invoke the transition it can go.